Modernize the Windows setup.

pkgw opened the pull request:
This updates the Windows stuff based on the experience I've gained building
the downstream X.org libraries this far. The key new bit is that it looks like
we're really supposed to use $LIBRARY_PREFIX as our install prefix rather than
$PREFIX. I also run autoreconf which looks like it will be necessary for the
Windows builds. Not actually needed for this package since it just installs
macro files, but I want the build scripts to be as consistent as possible.

pkgw committed:
Modernize the Windows setup.

This updates the Windows stuff based on the experience I've gained building
the downstream X.org libraries this far. The key new bit is that it looks like
we're really supposed to use $LIBRARY_PREFIX as our install prefix rather than
$PREFIX. I also run autoreconf which looks like it will be necessary for the
Windows builds. Not actually needed for this package since it just installs
macro files, but I want the build scripts to be as consistent as possible.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

pkgw committed:
D'oh, $LIBRARY_PREFIX is "/" which causes problems

Namely, paths beginning with // which lead to issues with attempts to modify the root directory.

pkgw committed:
Fix autoreconf for Win32
